mods/e2140/content/ed/infantry/a01/rules.yaml

```
@@ -0,0 +1,36 @@
+ed_infantry_a01:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A01 Android
```

Use original name *Android A01*:

```suggestion
		Name: Android A01
```